test/paper_submenu_test.dart

```diff
@@ -0,0 +1,169 @@
+@TestOn('browser')
+library polymer_elements.test.paper_progress_test;
+
+import 'dart:html';
+import 'package:polymer_elements/paper_item.dart';
+import 'package:polymer_elements/paper_submenu.dart';
+import 'package:polymer_elements/paper_menu.dart';
+import 'package:polymer_elements/iron_collapse.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-menu/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  group('<paper-submenu>', () {
+    PaperMenu menu;
+    PaperSubmenu sub1, sub2, sub3;
+    IronCollapse collapse1, collapse2, collapse3;
+    PaperItem trigger1, trigger2, trigger3;
+
+    setUp(() {
+      menu = fixture('basic');
+
+      sub1 = menu.querySelectorAll('paper-submenu')[0];
```

Comment:
Probably best if we only do one `querySelectorAll` here and assign that to a local variable, then use that to set sub1, sub2, and sub3.